index.android.js

```diff
@@ -18,63 +14,32 @@ const convertPoint = (name, point) => {
       'Array type is deprecated.'
     );
   }
+  // TODO: Update Android native code to receive a {x, y} object, not an array
```

Comment:
shouldn't this `TODO` be fixed in the same PR so that android integration won't break?

Reply:
It won't (shouldn't!) break here. I didn't change the behavior, just added an in-code note to change the behavior whenever the next person touches Android code.